src/if_ole.cpp

```diff
@@ -6,7 +6,7 @@
  * Do ":help credits" in Vim to see a list of people who contributed.
  */
 
-#if defined(FEAT_OLE) && defined(WIN32) && defined(FEAT_GUI)
+#if defined(FEAT_OLE) && defined(WIN32) && defined(FEAT_GUI_MSWIN)
```

Comment:
This is not mandatory, but I think we can remove `defined(WIN32) &&` from this line.
It is redundant.

Reply:
Remove it.